src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -62,5 +74,26 @@ public void setAllowCreatingCaseBeforePaymentsAreProcessed(boolean allowCreating
         this.allowCreatingCaseBeforePaymentsAreProcessed = allowCreatingCaseBeforePaymentsAreProcessed;
     }
 
+    public String getSurnameMapping(String formType) {
+        return surnameMappings.get(formType);
+    }
+
+    public void setSurnameMappings(List<FormFieldMapping> surnameMappings) {
+        this.surnameMappings = surnameMappings.stream()
+            .collect(groupingBy(FormFieldMapping::getFormType, toList()))
+            .entrySet().stream()
+            .collect(toMap(
+                e -> e.getKey(),
+                e -> {
+                    if (e.getValue().size() > 1) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4498197)